scripts/build.fsx

```diff
@@ -18,6 +18,8 @@ let param = Lib.setup <| fun p ->
             { p.NuGetFeed with 
                 ApiKey = environVarOrFail <| "NUGET_API_KEY" |> Some
             }
+        IncludeSymbols = false
+        EmbedSources = false

```

Comment:
Have you decided not to do "opt-out", as described in the issue, but go with "opt-in" as for apps?

Reply:
Maybe we should set defaults to `true` not here, but in `Fria2.Fake.Build` in `lib.fs`.